pippo-core/src/main/java/ro/pippo/core/Application.java

```diff
@@ -248,7 +249,8 @@ public final RequestResponseFactory getRequestResponseFactory() {
      * @return
      */
     protected RequestResponseFactory createRequestResponseFactory() {
-        return new RequestResponseFactory(this);
+//        return new RequestResponseFactory(this);
+        return new GZipRequestResponseFactory(this);
```

Comment:
Maybe there are people that don't want this feature builtin. From this reason I created `GZipRequestResponseFactory` that extends the simple `RequestResponseFactory`.
Also I want to mention that you can extends `GZipRequestResponseFactory` and to override `acceptsGZipEncoding` if you want more control (only some urls, ...).
What I don't like me is that I cannot use (yet) the wrapper/decorator pattern for `RequestResponseFactory`. For example if I want gzip compression on the fly I should use `GZipRequestResponseFactory`, if I want a different `SessionStorage` (for example Hazelcast) I should use `SessionResquestResponseFactory`. Now it's not possible to use both `GZipRequestResponseFactory` and `SessionResquestResponseFactory`.